packages/unmock-core/src/interfaces.ts

```diff
@@ -22,17 +24,18 @@ export interface IUnmockOptions extends ILogger {
 }
 
 export interface IBackend {
-  initialize(options: IUnmockOptions): States;
+  readonly services: IServiceStore;
+  initialize(options: IUnmockOptions): void;
   reset(): void;
 }
 
 export interface IUnmockPackage {
   allowedHosts: AllowedHosts;
-  on(): States;
-  init(): States;
-  initialize(): States;
+  services: IServiceStore;
+  on(): void;
```

Comment:
Any reason not to return `services` from `unmock.on()`? I kinda liked that even if it broke 
```js
beforeAll(() => unmock.on());
```

Maybe we should at some point distinguish between loading services and turning unmock on, those are quite different tasks and should be doable via different commands (of course we'd still expose a command that does both).

Reply:
No reason at all. I wasn't sure if it's best to return e.g. services or some other element of unmock.
I'll update it.

Also really agree on loading services as a different part, I think that should probably be in the constructor. Belongs in a different PR though.